.github/workflows/stale.yml

```diff
@@ -0,0 +1,16 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days'
+        stale-issue-label: 'stale'
+        days-before-stale: 90
```

Comment:
Just a suggested value. With 90, every issue will get a nudge every 3 months, and if its still relevant whoever is interested just has to comment with a "I still plan to do this" or "current state is ..." (or anything else, or just remove the label), and it will get another 90 days of being ignored.